fili-wikipedia-example/README.md

```diff
@@ -65,6 +65,51 @@ Here are some sample queries that you can run to verify your server:
 
       GET http://localhost:9998/v1/data/wikipedia/hour/?format=debug&metrics=count&dateTime=PT72H/current
 
+## Running Fili with Docker
+
+There is a [Docker image](https://hub.docker.com/r/mpardesh/fili/) for Fili which can be found on 
+Dockerhub. If you would like to experiment with Fili without having to download its dependencies, you can 
+[install](https://www.docker.com/community-edition) and start Docker. Then run these commands: 
+
+    
+    docker pull mpardesh/fili:1.0  
+    docker run --name fili-wikipedia-example -i --rm -p 3001:8081 -p 3000:8082 -p 9998:9998 mpardesh/fili:1.0
+    
+
+This will start a container. Please wait a few minutes for Druid to get ready. 
```

Comment:
While it's true that druid is up and running it still has to index the data which can take some time (I believe the druid script it fetching some data from S3)